src/cljs/netrunner/deckbuilder.cljs

```diff
@@ -508,7 +499,8 @@
 ;; Dot definitions
 (def zws "\u200B")                                          ; zero-width space for wrapping dots
 (def influence-dot (str "●" zws))                           ; normal influence dot
-(def mwl-dot (str "★" zws))                                 ; influence penalty from MWL
+(def banned-dot (str "✘" zws))                              ; on the banned list
+(def restricted-dot (str "🦄" zws))                          ; on the restricted list
```

Comment:
Love it!

Reply:
I can't take credit - I think I saw it used somewhere on Reddit first :)